.github/workflows/autobuild.yaml

```diff
@@ -122,26 +122,48 @@ jobs:
         "-DCMAKE_CXX_FLAGS_RELEASE=${{ matrix.config.cflags }}"
         ${{ matrix.config.cmake }}
     - name: Ninja
-      run: ninja -C ${{ env.CLANGD_DIR }} clangd
-    - name: Archive
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd clangd-indexer
```

Comment:
this builds on all platforms but only uploads for linux - not sure I quite see the point of that